lib/ui/global_widgets/default_button.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:news_pulse/core/constants/app_fonts.dart';
+import 'package:news_pulse/core/constants/dimens.dart';
+import 'package:news_pulse/core/di/di_manager.dart';
+
+class DefaultButton extends StatelessWidget {
+  const DefaultButton(
+      {required this.buttonText, required this.onPressed, super.key});
+  final String buttonText;
+  final Function() onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      // color: DIManager.findCC().white,
+      width: double.infinity,
+      margin: Dimens.defaultPageHorizontalPaddingSmall,
+      child: OutlinedButton(
+        style: OutlinedButton.styleFrom(
+          shape:
+              RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
```

Comment:
there is already a default Border Shape in app style and border-radius in dimens